packages/paste-website/src/pages/components/editable-code-block/index.mdx

```diff
@@ -0,0 +1,370 @@
+| Insert Tab character            | Tab                           | “Inserts the Tab character (or spaces depending on the indentation setting) and does not navigate to the next focusable element on the page.”                                                                 |
+| Toggle tab trapping             | Ctrl+Shift+M or Ctrl+M        | “Subsequent Tab keys will move focus out of the editor.”                                                                                                                                                      |
+| Go to Next Error or Warning     | F8                            |                                                                                                                                                                                                               |
+| Go to Previous Error or Warning | Shift+F8                      |                                                                                                                                                                                                               |
+| Show accessibility help         | Option+F1, Alt+F1, or Ctrl+F1 | Shows a “dialog…to find out the current position in the editor and to check the state of various accessibility options. The editor can be dynamically optimized for screen reader software from this dialog.” |
+
+## Examples
+
+### Default
+
+Use an Editable Code Block to allow users to enter code on multiple lines.
+To help users differentiate this from a static [Code Block](/components/code-block),
+Editable Code Block enables line numbers, code folding, and indentation guides by default.
+
+Put a logical label in an _Editable Code Block Header_ to describe what users need
+to put in the code block and/or the language used, similar to a form label.
+
+_Line numbers_ help users identify errors and cross-reference with existing code.
+Use line numbers when the expected input is longer than 5 lines.
+
+_Indentation guides and code folding_ help users read long blocks of
+code and troubleshoot errors.
+
+<StoryPreview
+  title="Default Editable Code Block"
+  storyID="components-editable-code-block--default"
+  code={`<EditableCodeBlockWrapper>
+  <EditableCodeBlockHeader>Typescript Example</EditableCodeBlockHeader>
+  <EditableCodeBlock
+    height="45vh"
+    defaultLanguage="typescript"
+    defaultValue={TypeScriptExample}
+  />
+</EditableCodeBlockWrapper>`}
+/>
+
+### Simple Editable Code Block
+
+Remove line numbers, indentation guides, and code folding from Editable Code Block only when:
+
+- Your page layout makes it visually clear to the user that the code block is editable, _and_
+- The expected code input is simple, like a short snippet of JSON.
+
+<StoryPreview
+  title="Simple Editable Code Block"
+  storyID="components-editable-code-block--no-special-features"
```

Comment:
issue: this preview and some other aren't working
![image](https://github.com/twilio-labs/paste/assets/75342690/4acf4cc6-67af-49ee-aefe-cddc5d8e4c69)

Reply:
This new storybook preview component only works after merge since it pulls the stories from the main branch storybook deploy